models/user/user_system.go

```diff
@@ -65,12 +65,46 @@ func (u *User) IsGiteaActions() bool {
 	return u != nil && u.ID == ActionsUserID
 }
 
+const (
+	ProjectWorkflowsUserID    int64 = -3
+	ProjectWorkflowsUserName        = "project-workflows"
```

Comment:
It seems to be a bad name and is likely to conflict with end user's name.

Reply:
All system bot type user have a `[bot]` prefix before the name and it should have no profile page. Then it's different from the user who have the same name.